plugwise/helper.py

```diff
@@ -122,14 +122,14 @@ def schemas_schedule_temp(schedules, name) -> float | None:
                 return schema_list[i][2]
 
 
-def types_finder(data):
+def types_finder(data) -> set:
     """Detect types within locations from logs."""
-    types = set()
+    types: set = set()
     for measure, attrs in HOME_MEASUREMENTS.items():
-        locator = f".//logs/point_log[type='{measure}']"
+        locator: str = f".//logs/point_log[type='{measure}']"
```

Comment:
Adding typing here is not needed, as it is already inferred
```suggestion
        locator = f".//logs/point_log[type='{measure}']"
```